tfs/src/main/java/hudson/plugins/tfs/TeamPluginGlobalConfig.java

```diff
@@ -108,6 +118,9 @@ public void setUserAccountMapper(UserAccountMapper userAccountMapper) {
     public boolean configure(StaplerRequest req, JSONObject json) throws FormException {

```

Comment:
is this called when the project is loaded in UI?

Reply:
When the global setting page is loaded